operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -504,7 +512,30 @@ func (dc *CassandraDatacenter) GetConfigAsJSON() (string, error) {
 	// cassandra.yaml whenever the seed nodes change.
 	seeds := []string{dc.GetSeedServiceName()}
 	seeds = append(seeds, dc.Spec.AdditionalSeeds...)
-	modelValues := serverconfig.GetModelValues(seeds, dc.Spec.ClusterName, dc.Name)
+
+	graphEnabled := 0
+	solrEnabled := 0
+	sparkEnabled := 0
+
+	if dc.Spec.ServerType == "dse" && dc.Spec.DseWorkloads != nil {
```

Comment:
👍  for checking the ServerType - how would you feel about adding a validation in the webhook too?

Reply:
I will do that.